test/test_time_extensions.rb

```diff
@@ -1,14 +1,14 @@
 require File.expand_path('../helper', __FILE__)
 
 describe "time extensions" do
-  
+
   before do
     BusinessTime::Config.currency_holidays = {
-      'USD' => ['2010-04-9'],
+      'USD' => ['2010-04-09'],
```

Comment:
We should definitely accept `'2010-04-9'`. If we do not, it should be considered to be a bug.